packages/lisk-transactions/test/utils/validation/validation.ts

```
@@ -247,6 +257,15 @@ describe('validation', () => {
 				return expect(validateAddress.bind(null, address)).to.throw(error);
 			});
 		});
+
+		describe('Given an address that has leading zeros', () => {
+			const address = '00015133512790761431L';
+			const error = 'Address number does not have natural representation.';
+
+			it('should throw', () => {
```

maybe `should throw an error` sounds better